docs/data-management.rst

```diff
@@ -130,6 +132,33 @@ Browsing and rendering
 
 #. You can revert to the original settings for an Image or Dataset. For example, using the context menu for a Dataset in the tree, select ``Rendering Settings > Set Imported and Save``.
 
+
+Manage Images in Dataset/Projects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can organize the data in the left-hand side tree by creating new Projects and Datasets. You can link the Images to the new or existing Datasets and Datasets to new or existing Projects.
+
+#. Use the Project |image2b| and Dataset |image2c| icons above the left-hand side tree or the right-click contextual menu to create new Datasets or Projects.
+
+#. Drag and drop Images between Datasets and Datasets between Projects.
```

Comment:
not limited to Dataset/Project since it can also be done with Plate/Screen. Maybe add a note

Reply:
fixed in https://github.com/ome/omero-guide-introduction/pull/5/commits/592c43f536e4835707f593d062df3b1d016f1ff7